ilastik/experimental/features.py

```diff
@@ -0,0 +1,62 @@
+class _Feature:
```

Comment:
this does not seem to handle 2D/3D features (3D data does not necessarily imply 3D features)
This means that the corresponding tests need some brushing up, too.